lib/src/segment.dart

```diff
@@ -53,5 +53,5 @@ class CircularChartSegmentTween extends Tween<CircularChartSegment> {
 
   @override
   CircularChartSegment lerp(double t) =>
-      CircularChartSegment.lerp(begin, end, t);
+      CircularChartSegment.lerp(begin!, end!, t);
```

Comment:
`begin` and `end` are required in the `CircularChartSegmentTween` initializer, but optional in the superclass, so it's safe to force-unwrap here (see line 49)